base/applications/mmc/console.c

```diff
@@ -311,6 +312,7 @@ FrameOnCommand(HWND hwnd,
 {
     PCONSOLE_MAINFRAME_WND Info;
     HWND hChild;
+    LPTSTR lpTitle;
```

Comment:
Why declare one variable for the whole function and the other one inside a block?

Reply:
Fixed